internal/services/containerapps/container_app_resource_test.go

```diff
@@ -1394,6 +1394,7 @@ resource "azurerm_container_app" "test" {
     external_enabled           = true
     target_port                = 5000
     transport                  = "http"
+    client_certificate_mode    = "accept"
```

Comment:
could we include an update to this property in the tests too?

Reply:
Sure. Will do.